xml/System.Web/HttpResponse.xml

```diff
@@ -2937,7 +2937,7 @@ Response.RedirectToRoutePermanent("Product",
       </ReturnValue>
       <Docs>
         <summary>Gets or sets the HTTP status code of the output returned to the client.</summary>
-        <value>An Integer representing the status of the HTTP output returned to the client. The default value is 200 (OK). For a listing of valid status codes, see [Http Status Codes](https://go.microsoft.com/fwlink/?LinkId=73157).</value>
+        <value>An Integer representing the status of the HTTP output returned to the client. The default value is 200 (OK). For a listing of valid status codes, see [Http Status Codes](https://docs.microsoft.com/en-us/dotnet/api/system.net.httpstatuscode).</value>
```

Comment:
```suggestion
        <value>An Integer representing the status of the HTTP output returned to the client. The default value is 200 (OK). For a listing of valid status codes, see [Http Status Codes](https://docs.microsoft.com/dotnet/api/system.net.httpstatuscode).</value>
```

Reply:
I managed to find the old target and fixed the FwLink internally to go to https://docs.microsoft.com/windows/win32/winhttp/http-status-codes. If we want to continue to send people there because the formatting is easier to read, I'd change like this:

```suggestion
        <value>An Integer representing the status of the HTTP output returned to the client. The default value is 200 (OK). For a listing of valid status codes, see [HTTP Status Codes](/windows/win32/winhttp/http-status-codes).</value>
```